frontend/mulighetsrommet-veileder-flate/src/views/tiltaksgjennomforing-detaljer/ViewTiltaksgjennomforingDetaljer.tsx

```diff
@@ -117,7 +118,7 @@ const ViewTiltaksgjennomforingDetaljer = ({
   const tilgjengelighetsstatusSomNokkelinfo: NokkelinfoProps = {
     nokkelinfoKomponenter: [
       {
-        _id: tiltaksgjennomforing.sanityId,
+        _id: tiltaksgjennomforing.sanityId || uuid(),
```

Comment:
Slett fra props i stedet for å legge til her

Reply:
Slettet